### ISSUE TITLE
Errors when syncing certain characters

### ISSUE THREAD

**Issue by kitschpatrol** [kitschpatrol/yanki-obsidian#13]
A note with the following content produces an error:

```markdown
Ułożenie znacznika w USG

---
doglowowo w strzałkowym;
w prawo w czołowym
(w lewo na plecach)
```

A note with the following content does not:

```markdown
Ulozenie znacznika w USG

---
doglowowo w strzałkowym;
w prawo w czołowym
(w lewo na plecach)
```

(As reported by @aesculapa)

**Comment by kitschpatrol** [comment]
I'm able to reproduce this. Getting the error:

```
Error: Write failed. File not found:
/Anki/Test/Ułożenie znacznika w USG.md
```

Special characters are only an issue in file names, not the note bodies — in this case the file is derived automatically from the contents.

It's Obsidian-specific, unicode file names work fine in the CLI version of Yanki.

Will fix but might be a few days.

**Comment by aesculapa** [comment]
Thank you for the update, Eric.

Great work identifying the issue. If there's anything further you need from me to assist in this matter, please let me know.
I'll wait patiently for the fix, no rush.

Best,

Aneta

**Comment by kitschpatrol** [comment]
Ok, this was a little tricky and seems to be related to [unicode normalization](https://www.w3.org/TR/charmod-norm/#unicodeNormalization) issues, where visually identical unicode strings have different byte representations.

Obsidian has its own ideas about normalization, as does Anki, as do Windows and macOS. I think I have a fix that works, so please upgrade to 1.4.3 and give it try.

**Comment by kitschpatrol** [comment]
I'm going to assume that this is fixed since the related tests are passing on my end and I haven't heard any updates to the contrary in this thread.

But please don't hesitate to reopen if you're still seeing this issue.
